doc/devel/coding_guide.rst

```diff
@@ -87,31 +84,67 @@ PR Review guidelines
   merge the PR and then open a new PR against upstream.
 
 
-Backports
-=========
+Branches and Backports
+======================
 
 
-When doing backports please include the branch you backported the
-commit to along with the SHA in a comment on the original PR.
+The current active branches are
 
-We do a backport from master to v2.0.x assuming:
+*master*
+  This will be Matplotlib 3.0.  Supports python 3.5+.
 
-* ``matplotlib`` is a read-only remote branch of the matplotlib/matplotlib repo
+*v2.2.x*
+  Maintenance branch for Matplotlib 2.2 LTS.  Supports python 2.7, 3.4+
+
+*v2.2.2-doc*
+  Documentation for the current release.
+
+
+We always will backport to 2.2.x
+
+- critical bug fixes (segfault, failure to import, things that the
+  user can not work around)
+- fixes for regressions against 2.0 or 2.1
+
+Everything else (regressions against 1.x versions, bugs/api
+inconsistencies the user can work around in their code) are on a
+case-by-case basis, should be low-risk, and need someone to advocate
+for and shepherd through the backport.
+
+Automated backports
+-------------------
 
-* ``DANGER`` is a read/write remote branch of the matplotlib/matplotlib repo
+We use meeseeksdev bot to automatically backport merges to the correct
+maintenance branch based on the milestone assigned to the PR.  The
+target branch is configured by putting ``on-merge: backport to
+TARGETBRANCH`` in the milestone description.
+
+The bot can also be manually triggered after a merge by leaving a
+message ``@meeseeksdev backport to BRANCH`` on the PR.
```

Comment:
note that only whitelisted devs can use that (and mention how to be added?) (or did that change?)

Reply:
There is no longer a whitelist.

Reply:
Can you document what to do when meeseeksdev replies with the cryptic error message "Something went wrong ... Please have a look at my logs." Are the logs somewhere visible?

Reply:
I think Mathias has to add you to another service to have access to the logs.